pkg/sqlite/filter.go

```diff
@@ -164,6 +169,23 @@ func (f *filterBuilder) addJoin(table, as, onClause string) {
 		table:    table,
 		as:       as,
 		onClause: onClause,
+		joinType: "LEFT",
```

Comment:
Just for naming clarity, I think `addJoin` should be renamed to `addLeftJoin`, because I would expect that a join without a left or right is an inner join by default.